etc/picongpu/taurus-tud/knl_picongpu.profile.example

```diff
@@ -0,0 +1,40 @@
+# Text Editor for Tools #######################################################
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load cmake/3.9.0
+module load git
+module load openmpi/2.1.0-intel
+
+# Compilers ###################################################################
+### ICC
+module load intel/2017.2.174
+# Boost needs to be build yourself for now!
+# These paths need to be adjusted.
+export LD_LIBRARY_PATH=<BOOST_INSTALLATION_DIR>:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=<BOOST_INSTALLATION_DIR>/lib:$LD_LIBRARY_PATH
+export BOOST_ROOT=<BOOST_INSTALLATION_DIR>
+
+export PICSRC=$HOME/src/picongpu
+export PIC_BACKEND="comp2b:native"
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+export PATH=$PATH:$PICSRC
+export PATH=$PATH:$PICSRC/src/tools/bin
```

Comment:
could you please add the alias to get an interactive node on hypnos-hzdr this is `alias getNode='qsub -I -q laser -lwalltime=00:30:00 -lnodes=1:ppn=64'`

Reply:
Okay